app/controllers/account/users_controller.rb

```diff
@@ -0,0 +1,73 @@
+class Account::UsersController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @users = current_tenant_users.page(params[:page]).per(10)
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
```

Comment:
rename - user_params => resource_params
it will be easier in future to use this controller as template for others